packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -410,7 +410,7 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
       // We're running field update under a transaction, so we can rollback if migration fails
       await fieldMetadataRepository.update(id, fieldMetadataForUpdate);
 
-      const updatedFieldMetadata = await fieldMetadataRepository.findOne({
+      const [updatedFieldMetadata] = await fieldMetadataRepository.find({
```

Comment:
Why this change?

Reply:
Totally decorelated from this PR. I should have made a side PR for this one. 

Reason : while pair prog, I remember @charlesBochet said `find` only cost 1 request instead of 2 with `findOne`